src/main/scala/JavaVersionCheckPlugin.scala

```diff
@@ -56,10 +56,12 @@ object JavaVersionCheck {
       def success(message: => String): Unit = realLog.success(message)
       def trace(t: => Throwable): Unit = realLog.trace(t)
     }
-    javac(sources = Nil,
-      classpath = Nil,
-      outputDirectory = file("."),
-      options = Seq("-version"))(captureVersionLog)
+    javac.run(
+      Array.empty,
+      Array("-help"),
+      IncToolOptions.create(java.util.Optional.empty(), true),
+      null,
+      captureVersionLog)
```

Comment:
This fails when i run `javaVersionCheck` with the Error:

```
sbt:Hello> javaVersionCheck
[error] java.lang.IllegalArgumentException: invalid flag: -help
[error] 	at com.sun.tools.javac.api.JavacTool.processOptions(JavacTool.java:206)
[error] 	at com.sun.tools.javac.api.JavacTool.getTask(JavacTool.java:156)
[error] 	at com.sun.tools.javac.api.JavacTool.getTask(JavacTool.java:107)
[error] 	at com.sun.tools.javac.api.JavacTool.getTask(JavacTool.java:64)
[error] 	at sbt.internal.inc.javac.LocalJavaCompiler.run(LocalJava.scala:152)
[error] 	at com.typesafe.sbt.JavaVersionCheck$.apply(JavaVersionCheckPlugin.scala:64)
[error] 	at com.typesafe.sbt.JavaVersionCheckPlugin$.$anonfun$javaVersionCheckSettings$2(JavaVersionCheckPlugin.scala:26)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:42)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:64)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:257)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:266)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:257)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:167)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:32)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[error] 	at java.lang.Thread.run(Thread.java:745)
[error] (*:javaVersionCheck) java.lang.IllegalArgumentException: invalid flag: -help
``` 

I couldn't find out how to solve that.

@eed3si9n  would it be an option to simplify this by just using `System.getProperty("java.version")` to determine the java version?